Snowflake.Data/Core/FileTransfer/StorageClient/SFGCSClient.cs

```diff
@@ -438,15 +362,13 @@ public void DownloadFile(SFFileMetadata fileMetadata, string fullDstPath, int ma
                         }
                     }
                     HandleDownloadResponse(response, fileMetadata);
+                    fileMetadata.resultStatus = ResultStatus.DOWNLOADED.ToString();
                 }
             }
             catch (WebException ex)
             {
                 fileMetadata = HandleDownloadFileErr(ex, fileMetadata);
```

Comment:
log missing

Reply:
Added logging in `HandleDownloadFileErr`